pyNN/nest/standardmodels/electrodes.py

```diff
@@ -155,3 +155,119 @@ class NoisyCurrentSource(NestCurrentSource, electrodes.NoisyCurrentSource):
     )
     nest_name = 'noise_generator'
 
+
+# Creation of native_electrode_type
+# See Issue 506
+def get_defaults(model_name):
```

Comment:
This is almost a direct copy of the function in `nest/cells.py`. Why copy it rather than importing from there?